### PR TITLE
Extractor and reverts

### PR DESCRIPTION
**Pull request by halfak**
Built off of https://github.com/wikimedia/articlequality/pull/127

### CODE REVIEW

File: articlequality/extractors/tests/test_ptwiki.py [hunk 1]
```diff
@@ -19,54 +19,149 @@ def __init__(self, title, namespace, revisions):
...
+                4, Timestamp(4), "aaa",
+                "{{Marca de projeto|1}}"  # Vandal messing up the template
+            ),
+            Revision(
+                5, Timestamp(5), "ccc",
+                "{{Marca de projeto|3}}"  # Rollback
+            ),
+            Revision(
+                6, Timestamp(6), "ddd",
+                "{{Marca de projeto|2}}"  # Active editor reevaluates the page
```
Comment (he7d3r) [inline articlequality/extractors/tests/test_ptwiki.py:134]: Supposedly, the sha1 of two revisions with the same content should be the same.
Therefore, shouldn't we change `{{Marca de projeto|2}}` to something else just to keep this implicit assumption valid, while keeping the same label?
E.g.: `{{Marca de projeto|2}}<!-- Reevaluated by John -->`

**Comment by he7d3r** [comment]
It is much more readable now :)

File: articlequality/extractors/tests/test_ptwiki.py (hunk 1 above)
Comment (halfak) [reply articlequality/extractors/tests/test_ptwiki.py]: {{done}}

File: articlequality/extractors/tests/test_ptwiki.py [hunk 2]
```diff
@@ -19,54 +19,149 @@ def __init__(self, title, namespace, revisions):
...
+                4, Timestamp(4), "aaa",
+                "{{Marca de projeto|1}}"  # Vandal messing up the template
+            ),
+            Revision(
+                5, Timestamp(5), "ccc",
+                "{{Marca de projeto|3}}"  # Rollback
+            ),
+            Revision(
+                6, Timestamp(6), "ddd",
+                "{{Marca de projeto|2}}<!-- re-evaluation -->"  # Active editor reevaluates the page
```
Comment (he7d3r) [inline articlequality/extractors/tests/test_ptwiki.py:134]: ./articlequality/extractors/tests/test_ptwiki.py:134:80: E501 line too long (100 > 79 characters)

**Pull request merged**